action.yml

```diff
@@ -35,6 +35,10 @@ inputs:
   compiled_modules:
     description: 'Whether to run tests with `compiled-modules`. For possible values, refer to https://docs.julialang.org/en/v1/manual/command-line-interface/#command-line-interface'
     default: 'yes'
+  threads:
+    description: 'Value passed to the --threads flag. Default value: auto.'
+    default: 'auto'
```

Comment:
`auto` isn't the julia default unfortunately, so this will need to be a bit cleverer. Basically not set the arg by default

Reply:
Ah, yeah.
This might just work:
`default: "${JULIA_NUM_THREADS:=1}"`

Or I can do a
```
if [[ -n ${{ input.threads }} ]]; then
   julia ... --threads ${{ input.threads }} -e ...
else
   julia ...
fi
```

Reply:
Better to populate a var for the threads arg and interpolate it in, as if any other args need the same behavior different if paths for the top level cmd would make for a mess of if statements

Reply:
Sorry, I meant a different approach like `threads_arg = inputs.threads == "default" ? "" : "--threads=${{ inputs.threads}}`

Then interpolate `threads_arg` into the cmd, making use of it being empty to avoid setting the arg